CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # go-filecoin changelog
 
+## go-filecoin 0.4.6
+
+This release includes the first steps towards increasing the performance for new nodes joining the network through the graphsync protocol.
+It also includes the new command `mining status` which allows users to understand the current state of their miner by providing the current proving period, as well as the current proving sector set.
+We've also shipped piece inclusion proofs which allow for storage clients to verify the inclusion of their data in the sealed sector reported back by the storage miner.
+
+### Features
+
+#### Chain sync
+
+Chain downloads have switched to [go-graphsync](https://github.com/ipfs/go-graphsync) for more efficient downloads, and we’ve also reduced block header size by moving messages and receipts out of the header.
+We’re expecting this to speed up chain sync & setup time.
```

Comment:
I would say *much* faster downloads.

The block header change is expected to significantly reduce memory requirements for a running node.